### DISCUSSION TITLE
Osu! lazer ios

### DISCUSSION THREAD

**Discussion by gabe577** [ppy/osu#15966] [Q&A] [answered]
Is the osu lazer ios build still out of operation?
(I apologize if this question has been asked recently)

**Comment by frenzibyte** [comment, marked as answer]
This has been asked a lot, you could've researched through the discussions thread and found it. iOS right now only lacks publishing new builds, which is tracked under https://github.com/ppy/osu/issues/15742.

**Comment by KrisCris** [comment]
Actually, you can build one by yourself if you want.

**Reply by frenzibyte** [reply]
I wouldn't suggest doing so unless whoever wants it have the necessary skills and tools to do it (and multiplayer access will be unavailable given it's a non-official build).

**Reply by KrisCris** [reply]
Then will this game eventually be on AppStore?

**Reply by smoogipoo** [reply]
Eventually, yes.
